nipype/interfaces/spm/base.py

```diff
@@ -171,7 +171,10 @@ def getinfo(klass, matlab_cmd=None, paths=None, use_mcr=None):
             If none of the above was successful, the fallback value of
             'matlab -nodesktop -nosplash' will be used.
         paths : str
+            Add paths to matlab session
```

Comment:
shouldn't we suggest that here should be also path to `spm` (that can be completely different than matlab) if we want to run spm interfaces ?

Reply:
IMO, that's clear from the function description: "Returns the path to the SPM directory in the Matlab path".